applications/dashboard/src/scripts/compatibilityStyles/buttonStylesCompat.ts

```diff
@@ -41,42 +41,64 @@ export const buttonCSS = () => {
     mixinButton(".BigButton:not(.Danger)", ButtonTypes.PRIMARY);
     mixinButton(".NewConversation.NewConversation", ButtonTypes.PRIMARY);
     mixinButton(".groupToolbar .Button.Primary", ButtonTypes.PRIMARY);
-    mixinButton(".BoxButtons .Button.Primary", ButtonTypes.PRIMARY);
+    mixinButton(".BoxButtons .ButtonGroup.Multi .Button.Primary", ButtonTypes.PRIMARY);
     mixinButton(".Section-Members .Group-RemoveMember", ButtonTypes.PRIMARY);
     mixinButton(".group-members-filter-box .Button.search", ButtonTypes.PRIMARY);
     mixinButton("#Form_Ban", ButtonTypes.PRIMARY);
     mixinButton(".Popup #UserBadgeForm button", ButtonTypes.PRIMARY);
     mixinButton(".Button.Handle", ButtonTypes.PRIMARY);
     mixinButton("div.Popup .Body .Button.Primary", ButtonTypes.PRIMARY);
     mixinButton(".ButtonGroup.Multi .Button.Handle", ButtonTypes.PRIMARY);
-    mixinButton(".ButtonGroup.Multi .Button.Handle .Sprite.SpDropdownHandle", ButtonTypes.PRIMARY);
+    // mixinButton(".ButtonGroup.Multi .Button.Handle .Sprite.SpDropdownHandle", ButtonTypes.PRIMARY);
```

Comment:
Usually we don't push commented code. Is it intentional here?

Reply:
Yeah this should go.
```suggestion
```